aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -170,29 +170,38 @@ These concrete rules structure our communications.
 
 ## Capitalization
 
-The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
-
-Sentences follow sentence capitalization rules:
-
+There are four types of capitalization. 
```

Comment:
There are more than four types.  There is "all caps", but we don't use it.  Also, camelCase usually refers to words where the first letter is NOT capitalized (hence, the camel reference).  PascalCase is the common term for when the first lettrer of each word is capitalized.

Reply:
Yes, I forgot all caps. I wrote this a little quickly. In the original version, which I did find, I had 5 lol

Reply:
I've edited this.